sw/device/silicon_creator/mask_rom/mask_rom.ld

```diff
@@ -23,6 +23,13 @@ INCLUDE hw/top_earlgrey/sw/autogen/top_earlgrey_memory.ld
  */
 _mask_rom_boot_address = ORIGIN(rom);
 
+/**
+ * Symbols to be used in the setup of the address translation for ROM_EXT.
+ */
+_rom_ext_boot_address = ORIGIN(eflash_rom_ext);
```

Comment:
We should already have this information since we are able to locate the manifests for both slots.

Reply:
The name is representing right, this address is the Virtual address (VMA). The address we have and matches with the manifest is the physical address (LMA).